openvasd/openvasd.c

```diff
@@ -285,6 +284,8 @@ openvasd_send_request (openvasd_connector_t conn,
       conn->stream_resp = g_malloc0 (sizeof (struct gvm_http_response_stream));
     }
 
+  gvm_http_headers_t *custom_headers = init_customheader (conn->apikey, TRUE);
```

Comment:
I think moving the custom headers here is a good idea but I think the second parameter of `init_customheader` for setting the content type should depend on whether the data is empty or not.

Reply:
Oh shoot I missed that. Fixed in [c2d6221](https://github.com/greenbone/gvm-libs/pull/908/commits/c2d6221ee350d304b1b5510ea7825075781e1da8).